Add bell hooks feminism is for everybody study guide

lizgene opened the pull request:
## Why?

Another study guide!

## What?

Added the bell hooks _Feminism is for Everybody_ study guide.

## How can/should this be manually tested?

[Local URL](http://127.0.0.1:4000/sf-feminist-book-club/feminism-is-for-everybody-bell-hooks/)

## Screenshots (if appropriate)

![image](https://cloud.githubusercontent.com/assets/4473327/25032412/c26aee8e-2088-11e7-994c-e2dd004396ce.png)

Commit:
Add bell hooks feminism is for everybody study guide

dashosaur commented on _posts/2017-04-13-bell-hooks-feminism-is-for-everybody.md:
```diff
@@ -0,0 +1,177 @@
+date:   2017-04-13
+permalink: "/feminism-is-for-everybody-bell-hooks/"
+categories: feminist-book-club
+tags: feminism anti-racism anti-capitalism study-guide
```
why is study-guide a tag? aren't all the posts study guides?

lizgene replied:
Yeah idk, it was like, maybe I'd use this... someday... but not now... lol  

_And so the cruft begins_

Will remove 👍

pamo replied:
Study guide should be a category. See changes introduced by #22

dashosaur replied:
I think categories should be "study-guides feminist-book-club". Or maybe in reverse order?

pamo commented on _posts/2017-04-13-bell-hooks-feminism-is-for-everybody.md:
Can you double check that these don't need an extra line break?

lizgene replied:
I'll add it in regardless for style points 👍

pamo commented on _posts/2017-04-13-bell-hooks-feminism-is-for-everybody.md:
Try removing [_base.scss#L179](https://github.com/radicalbarracuda/sf-feminist-book-club/blob/master/_sass/_base.scss#L179) to get the line-height to adjust properly. Didn't realize I left this line.

Commit:
Changes from dev review

- Fix newline spacing of Markdown content in post
- Fix SASS issue with line spacing in blockquotes
- Add study-guides category
- Remove feminist-book-club category

lizgene commented:
@dashland @pamo comments addressed in commit https://github.com/radicalbarracuda/sf-feminist-book-club/pull/23/commits/43aa9c1423bb9efb342a02f855b852168968255e

pamo commented on _posts/2017-04-13-bell-hooks-feminism-is-for-everybody.md:
```diff
@@ -0,0 +1,204 @@
+---
+layout: posts
+title: "Feminism is for Everybody by bell hooks"
```
Does bell hooks prefer lower case a-la ee cummings?
![image](https://cloud.githubusercontent.com/assets/784889/25033846/4df00964-2096-11e7-818a-578a19a89566.png)

lizgene replied:
Yes indeed!

pamo commented on _sass/_base.scss:
```diff
@@ -172,11 +172,10 @@ table {
 
 blockquote {
-  @include type-setting(2);
+  @include type-setting(1);
```
We might also want to get rid of [L193-196](https://github.com/radicalbarracuda/sf-feminist-book-club/pull/23/files#diff-fd4aa8862bb3db5bc523e0b0e9b6084aR194) because of this 
![image](https://cloud.githubusercontent.com/assets/784889/25033903/111241d2-2097-11e7-9e33-abc516db65c0.png)

lizgene replied:
Updated 👍 

Related, I also requested some margin on the footer: https://github.com/radicalbarracuda/sf-feminist-book-club/pull/25

Commit:
Remove 0 margin on last child of blockquotes
